frontend/src/app/document/entity-legend/entity-legend.component.html

```diff
@@ -0,0 +1,3 @@
+<div *ngFor="let entity of entities">
+    <span class="dot" [ngClass]="entity | formatEntityClass"></span><span>{{entity}}</span>
+</div>
```

Comment:
Can you think of a way to make the different types of entities readable to colorblind users?

Reply:
I chose a colourblind-friendly palette. Do you think the colour differences would be too subtle for highlights?

Reply:
I used a colorbrewer from adobe to come up with colours which might be suitable for highlights (the original colour palette was borrowed from the visualization, but for text background they don't work too well), and came up with this:
<img width="174" alt="Screenshot 2024-06-20 at 17 33 23" src="https://github.com/UUDigitalHumanitieslab/I-analyzer/assets/11174072/eee15099-b7b3-4dc2-afbe-0404a049cd0c">

Reply:
As far as I know, colorblind-friendly palettes are useful for the most common forms of colorblindness, but it's impossible to come up with a set of colors that works for all visually impaired users, which is why it's recommended that color is never the _only_ means to distinguish an element - see [WCAG 2.1 1.4.1](https://www.w3.org/TR/WCAG21/#use-of-color).

In this case, perhaps the component could also use icons to indicate the type of entity?

Reply:
A solution I've come up with now: using flags at the start of the highlight to indicate the entity type:
<img width="554" alt="Screenshot 2024-06-24 at 13 28 01" src="https://github.com/UUDigitalHumanitieslab/I-analyzer/assets/11174072/066867ab-780b-4a19-94ea-7bbc625a4d95">

Alternatives I've considered:
- striping the highlights in different ways - this I thought would make the foreground text less visible
- using text balloons so users could hover to understand the type - this would make the interface less useable to color-blind users, as they cannot have the "at a glance" overview of which entities were found, and might also obstruct other users with pop-ups while reading the text
- using superscript icons after the highlight - I think superscripts would interrupt textflow more